app/_data/docs_nav_kuma_dev.yml

```diff
@@ -126,21 +137,65 @@ items:
             url: /production/upgrades-tuning/fine-tuning/
       - text: Init containers
         url: /production/init-containers/
-  - title: Explore
+      - text: Control Plane Configuration
+        url: /documentation/configuration/
+        items:
+          - text: Modifying the configuration
+            url: /documentation/configuration/#modifying-the-configuration
+          - text: Inspecting the configuration
+            url: /documentation/configuration/#inspecting-the-configuration
+          - text: Store
+            url: /documentation/configuration/#store
+  - title: Using Kuma
```

Comment:
Should we use?
```suggestion
  - title: Using {{site.mesh_product_name}}
```

Reply:
I'm not sure we are able to use templates in this file

Reply:
Yes it doesn't we need to do a manual change in the sidebar overrides